apps/tangle-dapp/types/bridge.ts

```diff
@@ -85,5 +96,17 @@ export type BridgeQueueTxItem = {
   tokenSymbol: string;
   creationTimestamp: number;
   state: BridgeTxState;
+  type: BridgeType;
   explorerUrl?: string;
 };
+
+export type BridgeFeeType =
+  | 'gas'
+  | 'bridge' // used with Sygma
+  | 'interchain'; // used with Hyperlane
```

Comment:
Maybe rename the types to `sygmaBridge` and `hyperlaneInterchain`? Also, in case that we don't have a specific need for strings, we should prefer using enums instead of string sum types, since enums are really easy to refactor, while strings may need manual adjustment.